atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -144,11 +144,6 @@ public int schemaMutationTimeoutMillis() {
     }
 
     @Value.Default
-    public int heartbeatTimePeriodMillis() {
```

Comment:
We added an ADR for heartbeat with the last PR. Do you feel we need something else added to that?